app/views/guide/process_application.html.slim

```diff
@@ -127,6 +127,9 @@ header
       li bound by a restrictions of proceedings order (RPO) that limits the cases that can be brought by the applicant 
       li someone other than the person who must pay the fee; third parties may not apply for help with fees unless they're acting as a litigation friend, parent or guardian   
     br
+    h5 id="not-eligible" If the applicant is not eligible to apply 
```

Comment:
This paragraph should go below 'Check the fee:' because if sums up the whole section. E.g. if the application is for an MCOL case, they are not eligible to apply.